README.md

```diff
@@ -45,10 +48,26 @@ Then you can:
     - lib
     - components
     - request
+- types
 - tests
+- config
+- tools
 - demos
+- client
+    - src
+    - types
 ```
 
+## Configuration
+
+You can configure the following values by either environment variables or configuration files:
+- KEYGUARD_ALLOWED_ORIGIN: The origin from which requests are accepted. '*' allows all origins. Be aware that slashes have to be masked by `\`. Defaults to 'https://accounts.nimiq-testnet.com'.
+- KEYGUARD_CDN: The CDN (content delivery network) from which the core library is served. Defaults to 'https://cdn.nimiq-testnet.com'.
+
+The most straight forward way is to use a configuration file, which has to be placed in `config` folder, and pass their name as argument to the build script. `yarn build local` uses `local.conf`. Some sample files are provided.
+
+You can also set those values in your server configuration via environment variables. Please refer to your server's configuration, e.g. [https://httpd.apache.org/docs/2.4/env.html] for Apache.
```

Comment:
I don't quite understand why I would set them in my server. Or do you mean local development server? Because those env variables are not read during runtime, only during build, right?

Reply:
Exactly, it's a compile time config. So setting them in server configuration would make sense if you have a CI which builds on the server. I guess that should be clarified.

Reply:
updated readme